definitions/subworkflows/qc_exome_no_verify_bam.cwl

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Exome QC workflow"
+requirements:
+    - class: SchemaDefRequirement
+      types:
+          - $import: ../types/labelled_file.yml
+    - class: SubworkflowFeatureRequirement
+inputs:
+    bam:
+        type: File
+        secondaryFiles: [^.bai]
```

Comment:
Several tools in this PR were modified to expect both index names.  Should this one follow suit?

Reply:
I don't think so?  I have always been annoyed by the need to pass both for a handful of tools. None of these seem to require both, so I think it's fine.

Reply:
Yeah, certainly if they don't get used they're not necessary. I only asked because they both got added here for `pindel` and `mutect`, which surprised me.

Reply:
I don't think those are needed, and I can't recall adding them. I think they may have come as part of a merge with some stuff that @Matthew-Mosior did?  Matthew, any insight?

Reply:
Removed those changes, tested it, and I think it's all set now.